lib/Migrator/APIDiffMigratorPass.cpp

```diff
@@ -605,8 +605,11 @@ struct APIDiffMigratorPass : public ASTMigratorPass, public SourceEntityWalker {
                                           Arg->getStartLoc().getAdvancedLoc(1));
 
           // Replace "x.getY(" with "x.Y =".
-          Editor.replace(ReplaceRange, (llvm::Twine(Walker.Result.str().
-                                                   substr(3)) + " = ").str());
+          auto Replacement = (llvm::Twine(Walker.Result.str()
+                                          .substr(3)) + " = ").str();
+          Replacement[0] = tolower(Replacement[0]);
```

Comment:
It might be worth using `camel_case:: toLowercaseWord()`  from `swift/Basic/StringExtras.h` here, so we don't migrate `.getURL` to `.uRL`.